Formula/dmd.rb

```diff
@@ -47,6 +48,11 @@ class Dmd < Formula
   def install
     make_args = ["INSTALL_DIR=#{prefix}", "MODEL=#{Hardware::CPU.bits}", "-f", "posix.mak"]
 
+    # VERSION file is wrong upstream, has happened before, so we just overwrite it here.
+    version_file = (buildpath/"VERSION")
+    rm version_file
+    version_file.write version
```

Comment:
Have to remove first because otherwise it complains about overwriting an
existing file.
On 10 May 2016 9:33 am, "Mike McQuaid" <notifications@github.com> wrote:

> In Formula/dmd.rb
> <https://github.com/Homebrew/homebrew-core/pull/1040#discussion_r62630959>
> :
>
> > @@ -47,6 +48,11 @@ class Dmd < Formula
> >    def install
> >      make_args = ["INSTALL_DIR=#{prefix}", "MODEL=#{Hardware::CPU.bits}", "-f", "posix.mak"]
> >
> > +    # VERSION file is wrong upstream, has happened before, so we just overwrite it here.
> > +    version_file = (buildpath/"VERSION")
> > +    rm version_file
> > +    version_file.write version
>
> Does (buildpath/"VERSION").write version work?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew-core/pull/1040/files/b12f2d681b433bba2522f0651bad81141fc2b35e#r62630959>
>